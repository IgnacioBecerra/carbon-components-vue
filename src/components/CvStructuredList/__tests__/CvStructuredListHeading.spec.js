import { shallowMount } from '@vue/test-utils';
import { CvStructuredListHeading } from '../';

describe('CvStructuredListHeading', () => {
  it('should render correctly', async () => {
    const wrapper = shallowMount(CvStructuredListHeading);

    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should render with slot correctly', async () => {
    const wrapper = shallowMount(CvStructuredListHeading, {
      slots: {
        default:
          '<div class="headings-slot-default-test">Headings slot default test</div>',
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
