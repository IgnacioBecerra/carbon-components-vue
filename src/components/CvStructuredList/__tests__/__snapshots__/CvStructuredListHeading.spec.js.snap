// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CvStructuredListHeading should render correctly 1`] = `<div class="cv-structured-list-heading bx--structured-list-th"></div>`;

exports[`CvStructuredListHeading should render with slot correctly 1`] = `
<div class="cv-structured-list-heading bx--structured-list-th">
  <div class="headings-slot-default-test">Headings slot default test</div>
</div>
`;
